kotsadm/web/src/components/apps/AppConfig.jsx

```diff
@@ -131,10 +132,10 @@ class AppConfig extends Component {
   handleSave = async () => {
     this.setState({ savingConfig: true, configError: "" });
 
-    const { fromLicenseFlow, history, match } = this.props;
+    const { fromLicenseFlow, history } = this.props;
     const sequence = this.getSequence();
     const slug = this.getSlug();
-    const createNewVersion = !fromLicenseFlow && match.params.sequence == undefined;
```

Comment:
not string on JS here, but should it be `===` here and also is checking for `undefined` future-proof?